src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -177,7 +195,11 @@ public class GenerateProtoTask extends DefaultTask {
     checkInitializing()
     Preconditions.checkState(this.outputBaseDir == null, 'outputBaseDir is already set')
     this.outputBaseDir = outputBaseDir
-    outputs.dir outputBaseDir
+  }
+
+  @OutputDirectory
+  String getOutputBaseDir() {
```

Comment:
It seems like the `outputBaseDir` isn't the actual output, only a base location for the actual outputs. Each plugin would output in a sub directory in there. I suppose for now declaring all of the base dir as an output is fine.

Reply:
I think declaring this as an `@OutputDirectory` is a step in the right direction without changing the behavior. We can have followup PRs to make it more correct later.